### ISSUE TITLE
Checkboxes on the 'Assign Users' table are missing programmatic labels

### ISSUE THREAD

**Issue by SteelWagstaff** [pressbooks/pressbooks-multi-institution#341]
Checkboxes inside the table on the Assign Users pages are missing programmatic labels. 

<img width="512" height="131" alt="Image" src="https://github.com/user-attachments/assets/5d8bbff7-7189-4ae4-8b64-ddec9996d551" />

[Report Link](https://docs.google.com/document/d/1_fN8pR56TLmivnEY_IVLEQay-CYBo4z4/edit#bookmark=id.yjhh5l2b4w88)
